_format/1.1/index.md

```diff
@@ -2125,18 +2125,20 @@ Error objects provide additional information about problems encountered while
 performing an operation. Error objects **MUST** be returned as an array
 keyed by `errors` in the top level of a JSON:API document.
 
-An error object **MAY** have the following members:
+An error object **MAY** have the following members, and **MUST** contain at
+least one of:
 
 * `id`: a unique identifier for this particular occurrence of the problem.
-* `links`: a [links object][links] containing the following members:
+* `links`: a [links object][links] that **MAY** contain the following members,
+  and that **MUST** contain at least one of:
```

Comment:
I discussed this with @gabesullice and we're both uncomfortable including this MUST requirement here. Throughout the spec, `links` objects do not have any particular required members. It seems plausible that custom `links` could be included here without requiring `about` or `type` in addition.

Reply:
Good point; fixed in 0de92c05fb29e4240505b3b94e30bc6ab1fb2114.

Reply:
It would be great to make this more explicit. I always felt uncomfortable using custom link names in error objects because it was the only place where a links object had documented members. Maybe we can add something like 'may contain any member with the following members having a specified meaning'?

Reply:
Not sure what you mean about this being the only place where a links object has documented members; do self, related, and the pagination links not count?

Regarding "custom" links, maybe that's described somewhere in 1.1, but... I'm hesitant to approach that topic since I'm still trying to bring my implementation up to snuff for 1.0.

Reply:
@gabesullice and I were just talking about how custom members in `links` objects throughout the spec are still underspecified. We'd like to rectify this prior to 1.1 but, since it's a general issue, it doesn't need to hold up this PR.

Reply:
Good points, thanks!